graphql_compiler/compiler/ir_lowering_common/location_renaming.py

```diff
@@ -43,10 +50,10 @@ def translate_potential_location(location_translations, potential_location):
         return potential_location
 
 
-def make_location_rewriter_visitor_fn(location_translations):
+def make_location_rewriter_visitor_fn(location_translations: Dict[LocationT, LocationT]) -> Callable:
```

Comment:
`Callable` is insufficiently specific, I think we should be able to tighten it up by specifying what args it takes and what its return type is.